emukit/core/initial_designs/sobol_design.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+try:
+    from sobol_seq import i4_sobol_generate
+except ImportError:
+    raise ImportError('sobol_seq needs to be installed in order to use sobol design')
+
+from .base import ModelFreeDesignBase
+from .. import ParameterSpace
+
+
+class SobolDesign(ModelFreeDesignBase):
+    """
+    Sobol experiment design.
+    Based on sobol_seq implementation. For further reference see https://github.com/naught101/sobol_seq
+    """
+
+    def __init__(self, parameter_space: ParameterSpace) -> None:
+        """
+        param parameter_space: The parameter space to generate design for.
+        """
+        super(SobolDesign, self).__init__(parameter_space)
+
+    def get_samples(self, point_count: int) -> np.ndarray:
+        """
+        Generates requested amount of points.
+        :param point_count: Number of points required.
+        :return: A numpy array with shape (point_count x space_dim)
```

Comment:
This does not describe what is in the array

Reply:
Fixed there and in other model free design files